.github/workflows/CoverageCollection.yml

```diff
@@ -35,6 +35,16 @@ on:
         required: false
         default: '-r tests/requirements.txt'
         type: string
+      unittest_directory:
+        description: 'Path to the directory containing unit tests.'
+        required: false
+        default: 'tests/unit'
+        type: string
+      coverage_config:
+        description: 'Path to the .coveragerc file. Use pyproject.toml if empty.'
+        required: false
+        default: ''
```

Comment:
It seems that this is a breaking change because it requires existing workflows to explicitly declare `coverage_config: tests/.coveragerc`?

Reply:
I tried declaring it, but it is not handling it:

- https://github.com/edaa-org/pySVModel/runs/4627440518?check_suite_focus=true
- https://github.com/edaa-org/pySVModel/commit/5b8f42a661ef7fae572cba1b5427c63b54b57ff4

Reply:
So, the `.coveragerc` is not a valid TOML file. It should be:

```toml
[run]
branch = true

[report]
skip_covered = true
skip_empty = true

[html]
directory = '.cov'

[xml]
output = 'coverage.xml'
```

Reply:
With the fixed file format, it is read, but there is an issue with the variable name: https://github.com/edaa-org/pySVModel/runs/4627481390?check_suite_focus=true. It should be `coverageRCFile` instead of `pyProjectSettings`?

Reply:
After fixing the variable name and passing arg `-d` to `coverage html`, pySVModel works with this branch: https://github.com/edaa-org/pySVModel/runs/4627543532?check_suite_focus=true.

Reply:
The last use case would be testing this on a repo without `pyproject.toml` and without `.coveragerc`. Do we have any repo of that kind?

Reply:
Yes it's a little bit breaking. But I think it would be good to migrate towards pyproject.toml and reduce config files with maybe redundant information.

Reply:
All out projects either have that file, so we just need to handover the parameter or we transition immediately to pyproject.toml in such a case (preferred).

Reply:
Do you expect to force users to have coverage configuration data (either in `pyproject.toml` or in `.coveragerc`)?

Reply:
Hmmm good question. At least for me it makes almost no sense without them.
But users could have no entries and use defaults ...

Reply:
After pushing some additional fixes, this is tested without coverage configuration files as well:

- https://github.com/edaa-org/pySVModel/runs/4627738759?check_suite_focus=true
- https://github.com/edaa-org/pySVModel/commit/008bdf63ab4d2ea73e3bc4605a7ca7e805678b54